### ISSUE TITLE
click on disabled <b:commandButton> force a page reload

### ISSUE THREAD

**Issue by Kristover** [TheCoder4eu/BootsFaces-OSP#208]
When you click on < p:commandButton > with disabled="true", nothing appear. OK.
When you click on < b:commandButton > with disabled="true", a page reload occurs, which would imply some unwanted comportment

using BootsFaces 0.7.0
```xml
    <h:body onload="alert('onload');">
         <b:container fluid="true">
         <b:row><b:column col-lg="10">
         <h:form>
             <h:outputText value="Disabled TEST" />
             <b:commandButton disabled="true" value="test B" 
                               action="# {usersCtrl.doSomething}"  />
                                     
             <p:commandButton disabled="true" value="test P"
                               action="# {usersCtrl.doSomething}" />
         </h:form>
         </b:column></b:row>
         </b:container>
      </h:body>
```

**Comment by stephanrauh** [comment]
First of all, thank you very much for your detailed error description. This helps us a lot to trace down the error.

Actually, we've probably fixed the bug in version 0.8.0. See #142. Would you mind to use the developer snapshot of version 0.8.0 and tell us if the bug still occurs? You'll find detailed download instructions at #151.

**Comment by AlexanderBl4h** [comment]
I can tell, that the bug is solved in 0.8.0-SNAPSHOT. (At least for me!)

**Comment by stephanrauh** [comment]
OK, thanks for reporting back!
